file-updates/test.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env sh
+
+trap teardown EXIT
```

Comment:
Correct me if I'm wrong, but right now, without `set -e`, the script will always execute all the way to the end (unless there is an `exit` somewhere which doesn't seem to be the case). So I don't see a need for the `trap` - you could just call `teardown` at the end. But maybe we actually do want `set -e`?

Reply:
Since this is meant to run against a long lived instance (staging RHTAP), I think it's nice to clean up after testing in most cases. But I understand the debugging use case. How about we do the cleanup by default and skip it with a `--skip-cleanup` cli argument? Or is this something you would want in most cases anyway?

Either way, this is something that is not really required for this PR, feel free to defer this for later.

Reply:
added `--skip-cleanup` arg

Reply:
Thanks. See my new comment about this here: https://github.com/hacbs-release/release-integration-tests/pull/4/files#r1324157574